files/usr/share/cinnamon/cinnamon-looking-glass/cinnamon-looking-glass.py

```diff
@@ -209,28 +216,28 @@ def selectFile(self):
         return result
 
 class FileWatchHandler(pyinotify.ProcessEvent):
-    def __init__(self, view):
+    def myinit(self, view):
```

Comment:
this is the correct way to override ProcessEvent. you aren't supposed to override __init__